pegasus/sites.v3/hourofcode.com/views/home_signup.haml

```diff
@@ -1,24 +1,17 @@
 -# During Hour of Code season, show signup form and, unless it's pre-hoc, announcements. When it's not Hour of Code, hide signup form and announcements. Show "Sign Up is closed" message.
 -# We are hiding announcements in 2020
 - showAnnouncements = false
-%a#hocsignupform{name: 'signup'}
-#join
```

Comment:
Moved this anchor link here: https://github.com/code-dot-org/code-dot-org/blob/5394ebb3d381ad58309844568e6ccd66ef3442dc/pegasus/sites.v3/hourofcode.com/public/index.haml#L142